projects/can_api/dbc_generator.py

```diff
@@ -5,32 +5,31 @@
 
 __version__ = "2021.12.28"
 
+
 def main():
     """
     CLI to generate a DBC file from YAML files
     """
     parser = argparse.ArgumentParser(
-        description = "Generate DBC from YAML and DBC files",
+        description="Generate DBC from YAML and DBC files",
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
 
     parser.add_argument(
-        '-o', '--output',
-        default='out.dbc',
-        help='Output filename (default is out.dbc)'
+        "-o", "--output", default="out.dbc", help="Output filename (default is out.dbc)"
```

Comment:
most places you expand arguments onto separate lines while here you're condensing it onto one. Might be better to stay more consistent and do one or the other.